go.mod

```diff
@@ -17,15 +17,21 @@ require (
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/emicklei/go-restful/v3 v3.8.0 // indirect
+	github.com/evanphx/json-patch v4.12.0+incompatible // indirect
```

Comment:
Please revert all changes related to go modules. As far as I can tell none of the code added by this PR should change dependencies in anyway. In Kubernetes projects we have to be very careful with every new dependency we add.

Reply:
It does change dependecies, it adds 
```
	"k8s.io/cli-runtime/pkg/genericclioptions"
	"k8s.io/cli-runtime/pkg/printers"
	
```

@robscott  Please refer to my comment here https://github.com/kubernetes-sigs/ingress2gateway/pull/26#discussion_r1235612658

Reply:
Good catch, thanks!